conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -11,7 +11,18 @@ jobs:
     fetchDepth: {{ clone_depth }}
 {%- endif %}
   - script: |
-         rm -rf /opt/ghc
+         sudo mkdir -p /opt/empty_dir
+         for d in /opt/hostedtoolcache \
+                  /usr/lib/jvm \
+                  /usr/local/.ghcup \
+                  /usr/local/android \
+                  /usr/local/powershell \
+                  /usr/share/dotnet \
+                  /usr/share/swift \
```

Comment:
```suggestion
                  /usr/share/swift \
                  /opt/ghc \
```

Reply:
Yeah Billy noted [above]( https://github.com/conda-forge/conda-smithy/pull/1747#issuecomment-1646207903 ):

> The `/opt/ghc` directory no longer seems to exist so I removed that.

Think this is the only Azure Linux image we use

That said, there would be no harm in keeping the directory (since we ignore any no-ops when removing)

Reply:
Sure, no problem.